src/App.jsx

```diff
@@ -14,70 +16,74 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const defaultGoodValue = 'Jam';
+  const defaultGoodIndex = goods.indexOf(defaultGoodValue);
+  const [selectedGood, setSelectedGood] = useState(goods[defaultGoodIndex]);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {selectedGood ? (
+        <h1 className="title is-flex is-align-items-center">
```

Comment:
You have the same heading element `<h1 className="title is-flex is-align-items-center">` in both cases